docs/source/config.rst

```diff
@@ -172,6 +172,11 @@ dataset
 .. note :: 
   The DALI data loader configuration needs to be coupled with pytorch framework otherwise will fail.
 
+.. attention::
+  
+  For `format: jpeg`, it is not recommended to generate data due to its lossy compression nature. Instead, provide the path to original dataset in the `data_folder` parameter. 
+  More information on JPEG image generator is provided at :ref:`jpeg_generator_issue` section.
```

Comment:
Could u describe a bit here how the data folder needs to be structured. Add an examples with directory hierarchy

```
data_dir
├── train
│   ├── XXX.JPEG
│   ├── XXX.JPEG
├── val
│   ├── XXX.JPEG
│   ├── XXX.JPEG


```

Reply:
Can u point to that part directly here using a relative link. U can create a label and a link to it in rst.